h/search/index.py

```diff
@@ -26,7 +27,17 @@ class Window(namedtuple('Window', ['start', 'end'])):
     pass
 
 
-def index(es, annotation, request, target_index=None):
+def index(models_annotation, request):
+    search_annotation = persistence.Annotation.create(models_annotation)
+
+    request.registry.notify(AnnotationTransformEvent(
+        request, models_annotation, search_annotation.to_dict(),
+    ))
+
+    search_annotation.save()
```

Comment:
....Aaaannd it's that simple. If it wasn't for this annotation transform event notification we're sending, indexing an annotation would be nothing more than `persistence.Annotation.create(models_annotation).save)()`